modules/ml/document_store/faiss.py

```diff
@@ -36,45 +36,50 @@ def __init__(
-                                        The type is determined from the given string following the conventions
-                                        of the original FAISS index factory.
-                                        Recommended options:
-                                        - "Flat" (default): Best accuracy (= exact). Becomes slow and RAM intense for > 1 Mio docs.
-                                        - "HNSW": Graph-based heuristic. If not further specified,
-                                                  we use a RAM intense, but more accurate config:
-                                                  HNSW256, efConstruction=256 and efSearch=256
-                                        - "IVFx,Flat": Inverted Index. Replace x with the number of centroids aka nlist.
-                                                          Rule of thumb: nlist = 10 * sqrt (num_docs) is a good starting point.
-                                        For more details see:
-                                        - Overview of indices https://github.com/facebookresearch/faiss/wiki/Faiss-indexes
-                                        - Guideline for choosing an index https://github.com/facebookresearch/faiss/wiki/Guidelines-to-choose-an-index
-                                        - FAISS Index factory https://github.com/facebookresearch/faiss/wiki/The-index-factory
-                                        Benchmarks: XXX
-        :param faiss_index: Pass an existing FAISS Index, i.e. an empty one that you configured manually
-                            or one with docs that you used in Haystack before and want to load again.
-        :param return_embedding: To return document embedding
-        :param update_existing_documents: Whether to update any existing documents with the same ID when adding
-                                          documents. When set as True, any document with an existing ID gets updated.
-                                          If set to False, an error is raised if the document ID of the document being
-                                          added already exists.
-        :param index: Name of index in document store to use.
-        :param similarity: The similarity function used to compare document vectors. 'dot_product' is the default sine it is
-                   more performant with DPR embeddings. 'cosine' is recommended if you are using a Sentence BERT model.
+        Attributes:
+            sql_url (str, optional): SQL connection URL for database. It defaults to local file based SQLite DB (Change to PostgredSQL). For large scale
+                                     deployment, Postgres is recommended. Defaults to "postgresql+psycopg2://".
+            index_buffer_size (int, optional): When working with large datasets, the ingestion process(FAISS + SQL) can be buffered in
+                                               smaller chunks to reduce memory footprint. Defaults to 10000.
```

Comment:
Example:
```suggestion
            index_buffer_size:
                When working with large datasets, the ingestion process(FAISS + SQL) can be buffered in
                 smaller chunks to reduce memory footprint. Defaults to 10000.
```

Types could be ignored since they are stated in the method definition.